pkg/api/handlers/libpod/secrets.go

```diff
@@ -16,9 +18,17 @@ func CreateSecret(w http.ResponseWriter, r *http.Request) {
 		runtime = r.Context().Value("runtime").(*libpod.Runtime)
 		decoder = r.Context().Value("decoder").(*schema.Decoder)
 	)
+
+	decoder.RegisterConverter(map[string]string{}, func(str string) reflect.Value {
+		res := make(map[string]string)
+		json.Unmarshal([]byte(str), &res)
+		return reflect.ValueOf(res)
+	})
```

Comment:
@trusch NIT, can you add a comment that if this coventer code is needed elsewhere it should be moved to pkg/api/handlers/decoder.go. Thanks.